src/BattleProtos.cs

```diff
@@ -122,49 +122,49 @@ public static void AddProtos()
                 "真是的，我为什么要局限于现有的武器技术啊。我可是伊卡洛斯啊，能穿越黑洞，能超高速硬着陆，能去岩浆里泡澡去硫酸里游泳，我为什么不能大胆开发点离谱的武器？" +
                 "反正也是闲着，不如给虫子看看我的真本事！TNND，杀又杀不完，躲也躲不掉，还一直拖慢我的进度，跟我玩阴的是吧？直接来吧！");
-            ProtoRegistry.RegisterString("彩蛋7描述", "Sure enough, nothing will stop me if I really want to do. I was too limited before, as I can build a Dyson sphere, why does the cannon need bullets? Just Phaser fission! " +
+            ProtoRegistry.RegisterString("彩蛋7描述", "Sure enough, nothing will stop me if I really want something. I was too limited before, as I can build a Dyson sphere, why does the cannon need bullets? Just Phaser fission! " +
                 "Everything is good now, I don't need to be distracted by making bullets, so nice!\n" +
-                "By the way, I found <color=#c2853d>the intensity of attack has an upper limit. I really thought it will would infinitely...</color> Now it's so easy for me to defend them! " +
-                "I don't know if uploading universe matrix or building Dyson sphere will change anything, let me find at that time.", 
+                "By the way, I found <color=#c2853d>the intensity of attack has an upper limit. I really thought it would increase infinitely...</color> Now it's so easy for me to defend them! " +
+                "I don't know if uploading universe matrix or building Dyson sphere will change anything, let me find out.", 
                 "果然只要我出手没什么办不到的，之前还是太局限了，戴森球都能造出来了为什么武器还需要弹药？直接相位裂解就完事了，要不是降维技术不可逆我都有心想丢二向箔过去。这下好啦，不用专门分心去造子弹了，我要继续去当第四天灾了。" +
                 "顺便说一句，这些虫子可比索林原虫差远了，虽然索林原虫也是渣渣~~日记。\n" +
                 "另外我发现，<color=#c2853d>强度攻击原来是有上限的啊，我还真以为是无限提升呢...</color> 现在已经游刃有余了嘿嘿... 不知道给主脑上传宇宙矩阵或者建成戴森球后会不会有什么额外影响，等到那一步再看吧。");
-            ProtoRegistry.RegisterString("彩蛋8描述", "Young Icarus, no matter how you come here, you have proved your smart and power. Even facing unknown risks, you still light up your stars. " +
-                "Now you can proudly say, \"I've seen things that you absolutely can't believe, I've seen wormholes established in galaxies, I've seen fission rays flickering in swarms. All these moments will eventually fade away with time, like tears disappear in the rain.\" " +
-                "Continue your journey now! No matter what difficulties you encounter, don't be afraid and face them with a smile! Remember, you are a brave Icarus!", 
+            ProtoRegistry.RegisterString("彩蛋8描述", "Young Icarus, no matter how you came here, you have proved your smarts and strength. Even facing unknown risks, you still light up the stars. " +
+                "Now you can proudly say, \"I've seen things that you absolutely can't believe, I've seen wormholes established in galaxies, I've seen fission rays flickering in swarms. All those moments will be lost in time, like tears in rain.\" " +
+                "Continue your journey now! No matter what difficulties you encounter, don't be afraid and face them with a smile! Remember, you are the mighty Icarus!",
```

Comment:
I really like the prose in the diaries -- it's very evocative and fun to read. I tried not to mess up the tone as I really enjoyed it when I was playing